sw/device/lib/dif/dif_clkmgr.h

```diff
@@ -40,6 +40,26 @@ typedef uint32_t dif_clkmgr_gateable_clock_t;
  */
 typedef uint32_t dif_clkmgr_hintable_clock_t;
 
+/**
+ * Check if jitter is Enabled.
+ * @param clkmgr Clock Manager Handle.
+ * @param[out] is_enabled whether jitter is enabled or not.
+ * @returns The result of the operation.
+ */
+OT_WARN_UNUSED_RESULT
+dif_result_t dif_clkmgr_jitter_get_enabled(const dif_clkmgr_t *clkmgr,
+                                           bool *is_enabled);
```

Comment:
Shouldn't this return `dif_toggle_t`?

Reply:
I see dif_rstmgr's get_enabled functions use "dif_toggle_t *enable", but dif_clkmgr seems to use "bool *" pretty uniformly. I will replace them all.